src/screens/ContactScreen/index.js

```diff
@@ -129,14 +139,15 @@ const ContactScreen = ({navigation}) => {
       username={item.username}
       followed={extendedState.followed}
       userid={item.user_id}
+      isAnon={sourceScreen === ANONYMOUS}
       onPress={() => handleSelected(item)}
     />
   );
   // }
   const handleSelected = (value) => {
-    const copyFollowed = [...followed];
-    const copyUsername = [...usernames];
-    const copyUsers = [...selectedUsers];
+    const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
+    const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
+    const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is duplicating arrays only when `sourceScreen === ANONYMOUS`. However, it seems like these arrays are always duplicated later in the code. To avoid unnecessary duplication, you can move the array copying outside of the condition.
```diff
- const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
- const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
- const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
+ const copyFollowed = [...followed];
+ const copyUsername = [...usernames];
+ const copyUsers = [...selectedUsers];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->